include/GLFW/glfw3native.h

```diff
@@ -517,6 +517,24 @@ GLFWAPI EGLContext glfwGetEGLContext(GLFWwindow* window);
  *  @ingroup native
  */
 GLFWAPI EGLSurface glfwGetEGLSurface(GLFWwindow* window);
+
+
+/*! @brief Returns the `EGLConfig` of the specified window.
+ *
+ *  @return The `EGLConfig` of the specified window, or `EGL_NO_SURFACE` if an
+ *  [error](@ref error_handling) occurred.
+ *
+ *  @errors Possible errors include @ref GLFW_NO_WINDOW_CONTEXT and @ref
```

Comment:
If glfw wasn't initialized yet this could also return `EGL_NO_SURFACE`

Reply:
If GLFW isn't initialized yet (which I would consider as an error), this would indeed return `EGL_NO_SURFACE`. This is consistent with the behavior of `glfwGetEGLContext` and `glfwGetEGLSurface`.